Automattic/sensei
# Fix missing Fontawesome icons

Fixes #3133

### Changes proposed in this Pull Request

* Move `css/frontend/sensei.scss` into `css/frontend.scss`. 
* Some formatting.

The fontawesome icons are used both in the frontend and backend (`settings.scss`). Using relative paths for the font files in the output code is not handled well when they are included from different folders (`css/` and `css/frontend/`), and this is the easiest and cleanest solution/workaround.

### Testing instructions

* Ensure that icons work both in WP-admin and the frontend. 
   * Example pages: Sensei LMS > Grading > Grade a quiz,
   * Frontend: A course in progress (In progress button), or any notice, like opening a paid course page without being logged in


### Screenshot / Video

<img width="807" alt="image" src="https://user-images.githubusercontent.com/176949/83406172-5faea580-a40e-11ea-8969-ee3bd4eb5b77.png">
<img width="623" alt="image" src="https://user-images.githubusercontent.com/176949/83406208-7523cf80-a40e-11ea-8b34-8eb96d515d8e.png">
